### PR TITLE
Added patch to fix compilation with ncurses 6.3__2.13-3

### PR DESCRIPTION
**Pull request by abadawi591**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Added patch to fix compliation with ncurses 6.3.
version: n 2.12-3

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

File: SPECS-EXTENDED/powertop/powertop.spec [hunk 1]
```diff
@@ -69,6 +72,11 @@ touch %{_localstatedir}/cache/powertop/{saved_parameters.powertop,saved_results.
 %{_datadir}/bash-completion/completions/powertop
 
 %changelog
+
```
Comment (PawelWMS) [inline SPECS-EXTENDED/powertop/powertop.spec:75]:
```suggestion
```

**Review by PawelWMS — COMMENTED** [review]
Looks good, two small suggestions removing empty lines.

Please fill out the summary with asked information, thanks!

**Comment by abadawi591** [comment]
> Looks good, two small suggestions removing empty lines.
> 
> Please fill out the summary with asked information, thanks!

Done

**Review by oliviacrain — APPROVED** [review]
LGTM, thanks for the fix!

**Review by jslobodzian — APPROVED** [review]
Looks good!

**Pull request merged**